brave/brave-core
# Fixes tip format for media publishers

Resolves https://github.com/brave/brave-browser/issues/1976

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
- enable rewards
- go to youtube video
- do a tip
- go back to rewards settings page
- make sure that you see `on YouTube` in tip table

## Reviewer Checklist:
